src/consts/GraphqlActions.ts

```diff
@@ -14,7 +14,7 @@ export const GraphqlActions = {
   listWorkspaces: `query {
     workspacesList {
       items {
-        name id
+        name id region apiHost
```

Comment:
In the `GraphqlActions.ts` i see 3 kinds of style. I think will be good to refactor it and choose one of them. On the frontend we use the first one (each field on new line without comma)

``` 
workspaces {
  name
  workspace
}
```
```
items {
   name id region apiHost
}
```

```
deployStatus(buildName: $buildName) {
  status, message
}
```

Reply:
I agree with the first one